docs/Pivot-batchaggregator.md

```diff
@@ -17,7 +17,7 @@ of unique values in the field multiplied by the number of aggregates. When multi
 of columns is the product of unique values in each column multiplied by the number of aggregates. Maximum number of 
 pivot columns allowed are 2. Allowed column types are: int, long, double, float, string, boolean.
 
-**Pivot row:** Specifies a single field in the input schema. The unique values in this field will become the rows 
+**Pivot rows:** Specifies fields in the input schema. The unique values in this field will become the rows 
```

Comment:
should match the UI property
`Pivot Rows`

Reply:
Please make sure the documentation config and UI config names match.
For example `Default value` -> `Default Value`
`Number of partitions` -> `Number Of Partitions`..

Reply:
Missed ?